### PR TITLE
Rebuild webteam.canonical.com to become the webteam portal

### PR DESCRIPTION
**Pull request by tbille**
# Welcome to the webteam portal

- This site is the webteam portal. Is shows all the resources from the webteam
- It will be a site to share around and to help new comers to have a quick access to our tools that we build and host
- It will engage discussion in the discourse guides by making the docs more accessible
- Show to the world what we do as web developers & designers in Canonical
- It includes:
  - The release portal: http://0.0.0.0:8103/releases or https://webteam-canonical-com-49.demos.haus/releases (timeout on demo because of firewall  :slightly_frowning_face: )
  - The Guides from the team from discourse: http://0.0.0.0:8103/guides or https://webteam-canonical-com-49.demos.haus/guides
  - The list of people from the team (previously random.webteam.space): http://0.0.0.0:8103/team or https://webteam-canonical-com-49.demos.haus/team
  - An homepage with all the resources: http://0.0.0.0:8103 or https://webteam-canonical-com-49.demos.haus/
- Next step
  - Build CSS on the project (don't import vanilla directly from the asssets server)
  - Move practices to discourse and add them in the portal
  - Move tools.demo.haus to the portal under `/tools`
  - Optimise the launchapd team page
  - Tutorials for us??
  - Fix frontend built by me :smile: 

# QA

- `dotrun`
- Enjoy!

### CODE REVIEW

File: run [hunk 1]
```diff
@@ -1,615 +0,0 @@
-#! /usr/bin/env bash
```
Comment (jkfran) [inline run:1]: I really like removing this file from the repo but there are people still using this script (like Apple M1 users), should we remove it anyway?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
-    "serve": "./entrypoint 0.0.0.0:$PORT",
+    "start": "./entrypoint 0.0.0.0:$PORT",
```
Comment (jkfran) [inline package.json:4]: I think keeping both is neater even if in this case they do the same, `start` can be:
"start": "yarn run serve"

File: run (hunk 1 above)
Comment (tbille) [reply run]: You are right. Added back.

File: package.json (hunk 2 above)
Comment (tbille) [reply package.json]: :+1:  package.json updated

**Review by jkfran — APPROVED** [review]
Looks awesome 👍🏽

**Pull request merged**